package.json

```diff
@@ -5,6 +5,8 @@
   "license": "MIT",
   "scripts": {
     "build": "nx run-many -t build",
+    "branches:stale:dr": "DRY_RUN=true bash bin/clean-stale.sh",
```

Comment:
Can you explain to me why we need these scripts?

Reply:
We don't **need** them but it is for convenience. For my workflow I might have lots of branches and locally and once they are merged into main I no longer need them locally so they can be deleted. Rather than going one by one and deleting each local branch manually. This will only delete branches that no longer exist on origin aka "stale branches".

Reply:
Yeah, OK, I get that I usually don't commit those types of things and keep the scripts minimal. All good though, if you want to add it